go.mod

```diff
@@ -10,7 +10,7 @@ require (
 	github.com/goccy/go-yaml v1.12.0
 	github.com/google/go-cmp v0.6.0
 	github.com/google/uuid v1.6.0
-	github.com/kuadrant/authorino v0.18.0
+	github.com/kuadrant/authorino v0.18.1-0.20241030162053-82d7619acc40
```

Comment:
For now importing the version from main until we have a `v0.19.0` release

Reply:
Updated to use the [v0.19.0 release](https://github.com/Kuadrant/authorino/releases/tag/v0.19.0)